src/messages/en.json

```diff
@@ -1,3 +1,58 @@
 {
-    "home": "Welcome Home"
+    "404": "404",
+    "home": "Home",
+    "signout": "Signout",
+    "signup": "Signup",
+    "signin": "Signin",
+    "dashboard": "Dashoard",
+    "welcome-home": "Welcome Home",
+    "sign-up": "Sign up",
+    "page.signup.msg": "Let’s set up your account, Already have one?",
+    "sing-in-here": "Sign in here.",
+    "email": "Email",
+    "please-enter-your-email": "Please enter your email",
+    "password": "Password",
+    "please-enter-a-password": "Please enter a password",
+    "user-already-exist": "User already exist.",
+    "signup-privacy": "I accept the Projectify Terms of Service. For more information about Projectify's use and protection of your data, please see our Privacy Policy.",
+    "email-sent": "Email sent",
+    "i-sent-an-email-to": "I sent an email to",
+    "please-proceed-from-the-url-described-in-the-message": "Please proceed from the url described in the message.",
+    "top-page": "Top page",
+    "password-reset": "Password reset",
+    "page-user-reset-msg": "Please enter a your email to request a password reset.",
+    "user-not-found": "User not found.",
+    "send": "Send",
+    "password-reset-email-sent": "Password reset email sent",
+    "please-enter-your-email-to-request-a-password-reset": "Please enter your email to request a password reset.",
```

Comment:
Isn't this the same as the message above in line 23?

Reply:
yes I merged